skipchain/skipchain.go

```diff
@@ -698,6 +698,18 @@ func (s *Service) NewProtocol(ti *onet.TreeNodeInstance, conf *onet.GenericConfi
 	return
 }
 
+// AddClientKey can be used by other services to add a key so
```

Comment:
Where is the authentication on this? Why can't anyone who can talk on the websocket call this and add themselves?

Reply:
Because this is not exposed on the websocket and can only be called internally from one service to another.

One thing I'd like to do is some kind of Authentication-service, that can handle the authentication for all services. Now we have one for the identity, one for the skipchain and one for pop. And they don't all support {pin,public,pop} authentication. Which is crazy :(